pages/forgot-password.tsx

```diff
@@ -25,9 +26,15 @@ function ForgotPassword () {
         setErrorMessage("")
     }
 
-    console.log(email)
+    // Send a request to initiate the password reset process
+    // Send confirmation code to user's email
```

Comment:
Is this comment still supposed to be there? Where is the logic to send the confirmation code to a user's email?